packages/tanstack-react-router-urql/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@urql/tanstack-react-router",
+  "version": "1.0.0",
+  "description": "Convenience wrappers for using urql with SSR in @tanstack/react-router and @tanstack/start",
+  "sideEffects": false,
+  "homepage": "https://formidable.com/open-source/urql/docs/",
+  "bugs": "https://github.com/urql-graphql/urql/issues",
+  "license": "MIT",
+  "author": "Manuel Schiller",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/urql-graphql/urql.git",
+    "directory": "packages/tanstack-react-router-urql"
+  },
+  "main": "dist/urql-tanstack-react-router.js",
+  "module": "dist/urql-tanstack-react-router.es.js",
+  "types": "dist/urql-tanstack-react-router.d.ts",
```

Comment:
You are missing the export mappings, examples in other packages

Reply:
i was looking at `packages/react-urql/package.json` which also does not have export mappings.
Why do we need export mappings here?

Reply:
React-urql is a special case due to some old frameworks, vite is a good bundler so we should incorporate export mappings to future proof the package. What we do with react-urql is still a bit of a pickle, especially with regards to react-native